### PR TITLE
new alpha release for Gama

### PR DESCRIPTION
**Pull request by WoodenMaiden**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/121764)

### CODE REVIEW

**Comment by WoodenMaiden** [comment]
@microsoft-github-policy-service agree company="ACROSS (Advanced Computational Research On Sustainability Science)"

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

**Comment by stephengillie** [comment]
Hi @WoodenMaiden,

The installer isn't unattended. Is an installer switch missing?

**Comment by WoodenMaiden** [comment]
Hey @stephengillie :wave: 

This might be indeed a missing switch, however it is surprising to see this happening as our previous releases of GAMA didn't change anything in the innosetup script...

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/g/GamaPlatform/GamaAlpha/1.9.3-0a46eae/GamaPlatform.GamaAlpha.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  - gaml
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  Scope: user
+  InstallerUrl: https://github.com/gama-platform/gama/releases/download/1.9.3/GAMA_1.9.3_Windows_with_JDK_03.10.23_0a46eae.exe
+  InstallerSha256: BBB6F515D747858ED3196FCC87FEB7DDF8F9FF916030CE86FA7EC6194FC03D9A
+  InstallerSwitches:
+    Custom: /CurrentUser /Lang=english
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/g/GamaPlatform/GamaAlpha/1.9.3-0a46eae/GamaPlatform.GamaAlpha.installer.yaml:18]:
```suggestion
- Architecture: x64
  Scope: user
  InstallerUrl: https://github.com/gama-platform/gama/releases/download/1.9.3/GAMA_1.9.3_Windows_with_JDK_03.10.23_0a46eae.exe
  InstallerSha256: BBB6F515D747858ED3196FCC87FEB7DDF8F9FF916030CE86FA7EC6194FC03D9A
  InstallerSwitches:
    Custom: /CurrentUser /Lang=english
- Architecture: x64
```

**Comment by stephengillie** [comment]
I believe the word `explicit` in the description might have caused the [Policy 2.7](https://learn.microsoft.com/en-us/windows/package-manager/package/windows-package-manager-policies#27-adult-content) check.

**Review by stephengillie — APPROVED** [review]

**Pull request merged**